src/features/tags/components/TagsAside.js

```diff
@@ -25,27 +27,25 @@ class TagsAside extends Component {
 
     let visibleTags = activeTags === "new" ? mostRecentTags : topTags;
     let isHidden = activeTags === "" ? "hidden" : "";
-    let activeTop = activeTags === "top" ? "primary" : "secondary";
-    let activeNew = activeTags === "new" ? "primary" : "secondary";
+    let activeTop = activeTags === "top" ? "contained" : "text";
+    let activeNew = activeTags === "new" ? "contained" : "text";
 
     return (
       <>
-        <div className="button-group button-group--block">
-          <button
+          <Button 
```

Comment:
Were buttons used here instead of a button group because MUI's button group doesn't display an active button within a group? If so, MUI's [tabs](https://mui.com/components/tabs/#basic-tabs) would be a better option here